enterprise/j2ee.ejbcore/src/org/netbeans/modules/j2ee/ejbcore/action/CmFieldGenerator.java

```diff
@@ -22,11 +22,7 @@
 import com.sun.source.tree.ClassTree;
 import com.sun.source.tree.MethodTree;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
This is a problem.  It breaks the jdeps(1) dependency mapping. In addition, it kills the modularity of the project. 

For this reason, I have to give it a -1.  Can you please fix changes like this and resubmit.